avm/ptn/dev-ops/cicd-agents-and-runners/tests/e2e/private-devops-existing-vnet-aci/dependencies.bicep

```diff
@@ -0,0 +1,103 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Optional. The naming prefix.')
+param namePrefix string = 'dev'
```

Comment:
Should not be passed. Please create all names that may or may not conflict (usually top-level resource names like VNET, but subnet names) in the calling `main.test.bicep` like we do for all other test cases in AVM 😉

Reply:
Resolved